components/shared/Slider.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2006-present TrueChoice IP Holding Company, Inc.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of TrueChoice IP Holding Company, Inc.
+ * ("Confidential Information").  You shall not disclose such Confidential Information and shall
+ * use it only in accordance with the terms of the license agreement you entered into with the company.
+ */
+
+// dependencies
+// local files
+
+/**
+ * @description - returns input slider
+ * @param {string} props.sliderClassName - bootstrap class names for general slider layout
+ * @param {string} props.displayValueClassName - bootstrap class names for display value
+ * @param {function} props.handleInputChange - on change event handler
+ * @param {string} props.name - name to group the inputs into a category
+ * @param {string} props.value - what the slider is currently on
+ * @param {string} props.minValue - slider value lowest point
+ * @param {string} props.maxValue - slider value highest point
+ * @param {string} props.orient - sets slider to vertical position
+ * @return {jsx} - the Slider component to render
+ */
+const Slider = ({
+  sliderClassName,
+  displayValueClassName,
+  handleInputChange,
+  name,
+  value,
+  minValue,
+  maxValue,
+  orient
+}) => {
```

Comment:
add the `inputValue` and `onChange` functions from `Level.js` to this component instead

Reply:
Done!